docs/content/testnet/_index.en.md

```diff
@@ -1,18 +1,32 @@
+The TRISA TestNet has been established to provide a demonstration of the TRISA peer-to-peer protocol, host "robot VASP" services to facilitate TRISA integration, and is the location of the primary TRISA Global Directory Service (GDS) that facilitates public key exchange and peer discovery.  The TestNet instance is designed for testing, and the registration process is streamlined in the TestNet to facilitate quick integration. The Testnet enables you to test and validate transactions that share sensitive information safely and securely.
 
-{{% figure src="/img/testnet_architecture.png" %}}
+For reference, the TRISA Protocol documentation]({{< relref "getting-started/protocol">}} provides additional information on enabling the peer-to-peer exchange of compliance information.
 
 The TRISA TestNet is comprised of the following services:
 
 - [TRISA Directory Service](https://trisatest.net) - a user interface to explore the TRISA Global Directory Service and register to become a TRISA member
 - [TestNet Demo](https://vaspbot.net) - a demo site to show TRISA interactions between “robot” VASPs that run in the TestNet
 
-The TestNet also hosts three robot VASPs or rVASPs that have been implemented as a convenience for TRISA members to integrate their TRISA services. The primary rVASP is Alice, a secondary for demo purposes is Bob, and to test interactions with non-verified TRISA members, there is also an "evil" rVASP.
\ No newline at end of file
+The TestNet hosts three ["robot VASPs" (rVASPs)]({{< relref "testnet/rvasps">}}) that have been implemented as a convenience for TRISA members to integrate their TRISA services and validate the compliance solution safely. The primary rVASP is Alice, a secondary for demo purposes is Bob, and an "evil" rVASP to test interactions with non-verified TRISA members.
+
+The TestNet also provides a [command line utility]({{< relref "testnet/trisa-cli">}}) for interacting with the API for administrative and debugging purposes, using the testnet certificates.
+
+![TestNet Architecture](/img/testnet_architecture.png)
+
+## Joining the TestNet
+
+The following steps are required to join the TestNet:
+
+1. [Register](https://vaspdirectory.net/certificate/registration) with the GDS to create your TRISA Account with your VASP email address, where you can opt-in for TestNet Certificates. During registration, you can add collaborators within your organization.
+
+2. Complete the  multi-part VASP Verification form and due diligence process. Once approved, you gain will gain to the TestNet.
+
+3. Set up your TRISA node or integrate with a 3rd-party solution to start testing. When considering setting up your own server to host your own TRISA node, you must consider items necessary that may incur significant costs and resources, such as the server itself, long-term data storage solution, and developer time to configure and test.
```

Comment:
Reworded slightly plus fixed typo: "Set up your TRISA node or integrate with a 3rd-party solution to start testing. When considering setting up your own server to host your own TRISA node, you must consider resources necessary that may incur costs such as the server itself, a long-term data storage solution, and developer time to configure and test.